pytorch/jobs/docker/2.2/py3/Dockerfile.gpu

```diff
@@ -55,25 +51,14 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && apt-get clean
 
-# Installing Python3
-RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
```

Comment:
can we also get rid of the `PYTHON`, `PYTHON_PIP`, and `PYTHON_VERSION` variables above?